Specs/Data/Models/Box-Textured-KTX2-Basis/CesiumTexturedBoxTest.gltf

```diff
@@ -0,0 +1,210 @@
+{
```

Comment:
This should be updated to be spec-compliant - see [glTF specs](https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_texture_basisu).